pyfolio/tears.py

```diff
@@ -1553,7 +1553,7 @@ def create_perf_attrib_tear_sheet(returns,
 
     # aggregate perf attrib stats and show summary table
     perf_attrib.show_perf_attrib_stats(returns, positions, factor_returns,
-                                       factor_loadings, transactions)
+                                       factor_loadings, transactions, pos_in_dollars)
```

Comment:
you may want to pass this in as `pos_in_dollars=pos_in_dollars`, since `transactions` is also a keyword arg to `show_perf_attrib_stats`

Reply:
looks like flake8 failed as well, so put `pos_in_dollars=pos_in_dollars` on the next line